include/Song.h

```diff
@@ -274,6 +273,11 @@ class LMMS_EXPORT Song : public TrackContainer
 		return getPlayPos(m_playMode);
 	}
 
+	auto getTimeline(PlayMode mode) -> Timeline& { return m_timelines[static_cast<std::size_t>(mode)]; }
```

Comment:
I think we would just say `timeline` instead of `getTimeline`?

Reply:
True - I was going for consistency with `getPlayPos`, but we do prefer to omit the `get`.

Reply:
I think you may have forgotten about this, but it's fine 👍 the suggestion isn't really blocking anything.

Reply:
Ah yeah, I think it got mistakenly marked as resolved a while back and I never looked back up since. At least it's only a style thing.